packages/shared/src/components/sources/SourceCard.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import { useRouter } from 'next/router';
+import SquadMemberShortList from '../squads/SquadMemberShortList';
+import { Card } from '../cards/Card';
+import { Button } from '../buttons/Button';
+import { Connection } from '../../graphql/common';
+import {
+  SourceMember,
+  SourceMemberRole,
+  SourceType,
+} from '../../graphql/sources';
+import { Image } from '../image/Image';
+import { SquadDirectoryCardBannerDefault } from '../../lib/constants';
+import { SquadJoinButton } from '../squads/SquadJoinButton';
+import { Origin } from '../../lib/analytics';
+
+interface SourceCardProps {
+  title: string;
+  subtitle?: string;
+  icon?: ReactNode;
+  image?: string;
+  description?: string;
+  action?: {
+    text: string;
+    onClick?: () => void;
+  };
+  link?: {
+    text: string;
+    href: string;
+    target?: string;
+  };
+  members?: Connection<SourceMember>;
+  membersCount?: number;
+  permalink?: string;
+  id?: string;
+  banner?: string;
+  borderColor?: string;
+  type?: SourceType;
+  handle?: string;
+  memberInviteRole?: SourceMemberRole;
+  memberPostingRole?: SourceMemberRole;
+}
+
+export const SourceCard = ({
+  title,
+  subtitle,
+  icon,
+  image,
+  description,
+  action,
+  link,
+  members,
+  membersCount,
+  permalink,
+  id,
+  banner,
+  borderColor,
+  type,
+  handle,
+  memberInviteRole,
+  memberPostingRole,
+}: SourceCardProps): ReactElement => {
+  const router = useRouter();
+
+  const squad = {
+    name: title,
+    description,
+    members,
+    membersCount,
+    permalink,
+    id,
+    active: true,
+    public: true,
+    image,
+    handle,
+    memberInviteRole,
+    memberPostingRole,
+  };
+
+  return (
+    <Card
+      className={classNames(
+        'overflow-hidden p-0',
+        borderColor
+          ? `border-theme-color-${borderColor} hover:border-theme-color-${borderColor}`
```

Comment:
This is not allowed in Tailwind, you should always put fully typed classes else they won't render.
Best to create a custom pre-defined map for this (with our theme colors) I think we even have something like that.

[Background reading.](https://www.codeconcisely.com/posts/tailwind-css-dynamic-class/)